samples/25.message-reaction/src/main/java/com/microsoft/bot/sample/messagereaction/Application.java

```diff
@@ -0,0 +1,62 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+pakcage com.microsoft.bot.sample.messagereaction;
+
+import com.microsoft.bot.integration.AdapterWithErrorHandler;
+import com.microsoft.bot.integration.BotFrameworkHttpAdapter;
+import com.microsoft.bot.integration.spring.BotController;
+import com.microsoft.bot.integration.spring.BotDependencyConfiguration;
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Import;
+
+//
+// This is the starting point of the Sprint Boot Bot application.
+//
+@SpringBootApplication
+
+// Use the default BotController to receive incoming Channel messages. A custom
+// controller could be used by eliminating this import and creating a new
+// org.springframework.web.bind.annotation.RestController.
+// The default controller is created by the Spring Boot container using
+// dependency injection. The default route is /api/messages.
+@Import({BotController.class})
+
+/**
+ * This class extends the BotDependencyConfiguration which provides the default
+ * implementations for a Bot application.  The Application class should
+ * override methods in order to provide custom implementations.
+ */
+public class Application extends BotDependencyConfiguration {
+    public static void main(String[] args) {
+        SpringApplication.run(Application.class, args);
+    }
+
+    /**
+     * Returns the Bot for this application.
+     *
+     * <p>
+     *     The @Component annotation could be used on the Bot class instead of this method
+     *     with the @Bean annotation.
+     * </p>
+     *
+     * @return The Bot implementation for this application.
+     */
+    @Bean
+    public Bot getBot(ActivityLog log) {
+        return new MessageReactionBot(log);
+    }
```

Comment:
This is not the correct way to do this. 

Check the core-bot for [this](https://github.com/southworks/botbuilder-java/blob/external/feature/southworks/samples/13-core-bot/base/samples/13.core-bot/src/main/java/com/microsoft/bot/sample/core/Application.java#L58)

Reply:
I've been checking this. What you want me is to add a getter that returns the ActivityLog? I checked the [multilingual](https://github.com/southworks/botbuilder-java/blob/main/samples/17.multilingual-bot/src/main/java/com/microsoft/bot/sample/multilingual/Application.java#L56) and [qna](https://github.com/southworks/botbuilder-java/blob/main/samples/11.qnamaker/src/main/java/com/microsoft/bot/sample/qnamaker/Application.java#L50) bots and they don't have it. That's the change you want?

Reply:
Exactly, the bots you mentioned don't have the `ActivityLog`. The _similar scenario_ here is the core-bot. Apart from the `getter` method, it also needs to be called in the `getBot` in order to consume the `Activitylog`